client/public/index.html

```diff
@@ -3,13 +3,16 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
```

Comment:
I don't actually know anything about these changes. Pulled them from the `frontend-UI` branch when I was grabbing the correct favicon